driver_scripts/setup_ocean_region_groups.py

```diff
@@ -91,7 +90,7 @@ def remove_small_polygons(inFileName, outFileName, minArea):  # {{{
     write_all_features(features, outFileName, indent=4)  # }}}
 
 
-def build_basins():  # {{{
+def build_ocean_basins():  # {{{
     basinGroupName = 'OceanBasinRegionsGroup'
     basinFileName = 'oceanBasins.geojson'
```

Comment:
Okay, I went ahead and switched to a `Global Ocean` feature, consistent with the hard-wired feature.